src/FileCache.php

```diff
@@ -25,110 +57,124 @@ final class FileCache implements CacheInterface
     private const EXPIRATION_EXPIRED = -1;
 
     /**
-     * @var string the directory to store cache files. You may use [path alias](guide:concept-aliases) here.
-     */
-    private $cachePath;
-    /**
-     * @var string cache file suffix. Defaults to '.bin'.
-     */
-    private $cacheFileSuffix = '.bin';
-    /**
-     * @var int the level of sub-directories to store cache files. Defaults to 1.
-     * If the system has huge number of cache files (e.g. one million), you may use a bigger value
-     * (usually no bigger than 3). Using sub-directories is mainly to ensure the file system
-     * is not over burdened with a single directory having too many files.
+     * @var string The directory to store cache files. You may use path alias here.
```

Comment:
I don't see an ability to use aliases.

Reply:
It looks like alias resolving is done at config level so it has nothing to do with the cache itself.